LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -48,6 +48,7 @@ LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.</Descri
     <PackageIcon>icon.png</PackageIcon>
     <PackageReleaseNotes>https://code.videolan.org/videolan/LibVLCSharp/blob/master/NEWS</PackageReleaseNotes>
     <PackageTags>libvlc;vlc;videolan;native;c/c++;video;audio;player;media;mediaplayer;codec;ffmpeg;xamarin;graphics;ios;android;linux;windows;macos;cross-platform</PackageTags>
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
```

Comment:
It used to work without this. Why is this needed now?

Reply:
With the add of the `Generic.xaml` file, you need this to generate the `Generic.xbf` file (XAML binary format) and place it in the good subfolder. Without this, you would get an exception. In UWP class libraries, this option is set by default.
For the other librairies (Android, iOS, ...), there is no incidence.

Reply:
Ok.